File: backend/src/main/java/com/prefin/domain/money/AccountHistory.java
package com.prefin.domain.money;

import com.prefin.domain.user.Child;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AccountHistory {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "CHILD_ID")
    private Child child;

    private String transactionDate;

    private String transactionTime;

    @Column(columnDefinition = "VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci")
    private String briefs;

    private String deposit;

    private String withdraw;
}

File: mobile/Prefin/app/src/main/java/com/prefin/model/dto/AccountHistory.kt
package com.prefin.model.dto

data class AccountHistory(
    var briefs: String = "",
    var childId: Long = 0L,
    var deposit: String = "",
    var id: Long = 0L,
    var transactionDate: String = "",
    var transactionTime: String = "",
    var withdraw: String = "",
)
